From 93df35092506d428a0ed4402c865074803c221f1 Mon Sep 17 00:00:00 2001
From: Richard Liu <r66033@freescale.com>
Date: Sun, 27 Jan 2013 22:40:35 +0800
Subject: [PATCH 160/602] ENGR00241719 set DPI to 320 for HDMI dongle

set DPI to 320 for HDMI dongle to get large icons and font,
according to Google's design TV DPI should be 213, but we regard
HDMI dongle is a special case, we reference on market HDMI dongle
device they are customized and icons and font are large and we don't
want do do UI customize but just change DPI to 320 here to get large
icons and font.

Signed-off-by: Richard Liu <r66033@freescale.com>
---
 hdmidongle_6dq/init.rc | 5 ++++-
 imx6/hdmidongle_6dq.mk | 2 +-
 2 files changed, 5 insertions(+), 2 deletions(-)

diff --git a/hdmidongle_6dq/init.rc b/hdmidongle_6dq/init.rc
index 638a3ec..291a491 100755
--- a/hdmidongle_6dq/init.rc
+++ b/hdmidongle_6dq/init.rc
@@ -54,8 +54,11 @@ on boot
     setprop ro.sf.hwrotation 0
 
     # Set the density to 140dpi, default 128dpi is not good
-    setprop ro.sf.lcd_density 140
+    setprop ro.sf.lcd_density 320
 
+    # Set device type, tablet means use tablet layout no matter DPI and density
+    setprop sys.device.type tablet
+ 
 # change for FSL specific service
     chown system system /sys/devices/system/cpu/cpu0
     chown system system /sys/devices/system/cpu/cpu1
diff --git a/imx6/hdmidongle_6dq.mk b/imx6/hdmidongle_6dq.mk
index 613b584..a8f9c32 100644
--- a/imx6/hdmidongle_6dq.mk
+++ b/imx6/hdmidongle_6dq.mk
@@ -20,7 +20,7 @@ DEVICE_PACKAGE_OVERLAYS := device/fsl/hdmidongle_6dq/overlay
 
 PRODUCT_CHARACTERISTICS := tablet
 
-PRODUCT_AAPT_CONFIG += xlarge large tvdpi hdpi
+PRODUCT_AAPT_CONFIG += xlarge large tvdpi hdpi xhdpi
 
 PRODUCT_COPY_FILES += \
 	frameworks/native/data/etc/android.hardware.location.gps.xml:system/etc/permissions/android.hardware.location.gps.xml \
-- 
1.8.0

